samples/python/tutorial_code/core/file_input_output/file_input_output.py

```diff
@@ -0,0 +1,148 @@
+import cv2 as cv
+import numpy as np
+import sys
+
+def help(filename):
+    print (filename, 'shows the usage of the OpenCV serialization functionality.',
```

Comment:
Please use normal print from Python3 (+ `from __future__ import print_function` for Python 2.7).

Also it is better to use docstring.